executor.go

```diff
@@ -21,4 +21,25 @@ type CommandExecutor interface {
 
 // ExecuteCommandOption is an option that affects the behavior of a call to the
 // ExecuteCommand() method of the [CommandExecutor] interface.
-type ExecuteCommandOption struct{}
+type ExecuteCommandOption struct {
```

Comment:
This structure was just a placeholder, see `HandlerRoute` for an example of how we implement functional options in Dogma.

Reply:
Note that `HandlerRoute` _is_ the option type built by the functional options to the `ApplicationConfigurer.Routes()` method - it just doesn't have the `Option` suffix in this case.